npm install failure on `crocodile` not found

```
$ npm i
npm WARN deprecated lodash.assign@4.2.0: This package is deprecated. Use Object.assign.

> fsevents@1.0.14 install /Users/john/mysrc/crocodilejs/crocodile-cli/node_modules/fsevents
> node-pre-gyp install --fallback-to-build

[fsevents] Success: "/Users/john/mysrc/crocodilejs/crocodile-cli/node_modules/fsevents/lib/binding/Release/node-v48-darwin-x64/fse.node" is installed via remote

> crocodile@1.0.2 postinstall /Users/john/mysrc/crocodilejs/crocodile-cli
> crocodile

sh: crocodile: command not found

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/john/.nvm/versions/node/v6.7.0/bin/node" "/Users/john/.nvm/versions/node/v6.7.0/bin/npm" "i"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! crocodile@1.0.2 postinstall: `crocodile`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the crocodile@1.0.2 postinstall script 'crocodile'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the crocodile package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     crocodile
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs crocodile
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls crocodile
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/john/mysrc/crocodilejs/crocodile-cli/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -58,7 +58,8 @@
   "scripts": {
     "compile": "rm -rf ./lib && babel -d lib/ src/",
     "lint": "eslint .",
-    "postinstall": "crocodile",
+    "postinstall": "./bin/crocodile",
+    "preinstall": "npm run compile",
     "prepublish": "npm run test && npm run compile",
     "test": "npm run lint"
   }

--- a/package.json
+++ b/package.json
@@ -60,7 +60,6 @@
     "lint": "eslint .",
     "postinstall": "./bin/crocodile",
     "preinstall": "npm run compile",
-    "prepublish": "npm run test && npm run compile",
     "test": "npm run lint"
   }
 }

